spec/feature/icalendar_spec.rb

```diff
@@ -0,0 +1,53 @@
+RSpec.describe "Icalendar", type: :feature do
+  include CalendarHelpers
+
+  before do
+    ["/gwyliau-banc", "/bank-holidays"].each do |base_path|
+      content_item = { base_path:, schema_name: "calendar", document_type: "calendar" }
+      stub_content_store_has_item(base_path, content_item)
+      mock_calendar_fixtures
+    end
+  end
+
+  context "getting ICS version" do
+    before do
+      allow(File).to receive(:mtime).with(Rails.root.join("REVISION")).and_return(Time.zone.parse("2012-10-17 01:00:00"))
```

Comment:
Urgh what's this?! Maybe the spec pre-dates [TimeHelpers](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel_to). Can we use that instead?